### ISSUE TITLE
A11y_AzureDataStudio_ControlTab_ScreenReader :"View" link is not descriptive.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7674]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#WCAG2.4.4;DesktopApp;#NVDA;#JAWS;#SQL Azure Data Studio;#Win32
**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA , JAWS

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard->click on the "control tab " in the left nav bar-> navigate to the "Metricsdc" tab.
8. Start NVDA.
9. Navigate till "View " link using tab .
11. Observe the announcement by screen reader .

**Actual:**
Screen reader is only announcing the "View " link which is not descriptive.
the link description is not available to the users or any sort of reference.
**Similar issue for View Details button **
**Similar issue is also observed with JAWS .**
**Similar issue is also observed with the following screens:**
1)Big Data Cluster Overview page :
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
On this screen the issue is for "View Details".
 
**Expected:**
Screen Reader should read proper description for the link like what content the user would be getting or what is the link about.

**User Impact:**
Screen Reader should read proper description for the link like what content the user would be getting or what is the link about. These information should be conveyed for the user.

**Attachment for Reference:** 
[7674_A11y_AzureDataStudio_ControlTab_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3717474/7674_A11y_AzureDataStudio_ControlTab_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 167a9f991ae9ee0b9a61a4503ed4cda573aa59b2
Date: 2020-01-16T05:42:37.941Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
